src/detect.h

```diff
@@ -889,6 +889,9 @@ typedef struct DetectEngineCtx_ {
     /* maximum number of times a tx will get logged for a stream-only rule match */
     uint8_t stream_tx_log_limit;
 
+    /* force app-layer tx finding for alerts with signatures not having app-layer keywords */
+    uint8_t force_applayer;
```

Comment:
bool?

Reply:
right until we expand it to an enumeration